src/consensus/params.h

```diff
@@ -14,6 +14,7 @@ namespace Consensus {
 enum DeploymentPos
 {
     DEPLOYMENT_TESTDUMMY,
+    DEPLOYMENT_TAPROOT, // Deployment of taproot (BIPs 340-342)
```

Comment:
Unify Taproot capitalization in comments.
```suggestion
    DEPLOYMENT_TAPROOT, // Deployment of Taproot (BIPs 340-342)
```

Reply:
Updated.

Reply:
Btw, I wasn't planning to waste your time on this by suggesting to manually fix it up. You can just click apply suggestion on GitHub and it'll create a shared commit.